backend/kernelCI_app/management/commands/helpers/kcidbng_ingester.py

```diff
@@ -130,6 +130,64 @@ def consume_buffer(buffer: list[TableModels], item_type: TableNames) -> None:
     out("bulk_create %s: n=%d in %.3fs" % (item_type, len(buffer), time.time() - t0))
 
 
+def flush_buffers(
+    *,
+    issues_buf: list[Issues],
+    checkouts_buf: list[Checkouts],
+    builds_buf: list[Builds],
+    tests_buf: list[Tests],
+    incidents_buf: list[Incidents],
+) -> None:
```

Comment:
We could create a helper object that holds these lists, so we can just pass the object instead of passing 5 different lists